helevents/templates/rest_framework/event_list.html

```diff
@@ -100,6 +100,13 @@ <h3 id="event-text">Event publisher</h3>
 <pre><code>event/?publisher=ytj:0586977-6
 </code></pre>
 <p><a href="?publisher=ytj:0586977-6" title="json">See the result</a></p>
+<h3 id="event-text">Audience min and max age</h3>
```

Comment:
I would not include this in the main linkedevents documentation at the moment. That is because none of our actual event data have these ages now, so these filters are useless for linkedevents users. It is okay to include the fields in the code, though, because in the future they may come in handy.

This is one point in which you might need separate linked courses documentation that is geared towards courses users specifically, as only courses currently have this data available?